src/Axis.js

```diff
@@ -393,7 +393,7 @@ export default class Axis extends BaseClass {
 
     const labelHeight = max(textData, t => t.height) || 0;
 
-    if (horizontal) {
+    if (horizontal && typeof this._labelRotation === "undefined") {
```

Comment:
Set `this._labelRotate = true;` in the constructor, so then you don't need to check for `undefined`.